extension.yaml

```diff
@@ -157,10 +160,10 @@ params:
     immutable: true
 
   - param: MAILCHIMP_API_KEY
-    label: Mailchimp API key
+    label: Mailchimp OAuth Token
     description: >-
```

Comment:
I prefer the old description, personally.

"      What is your Mailchimp API key? To obtain a Mailchimp API key, go to your ..."

Reply:
@kevinthecheung Mailchimp would like all users to use an OAuth token instead of their API Key. The only way to get the token however is to complete the OAuth flow - it's not known beforehand unlike the API key. But that said, I'm Ok changing this and anything else; I don't have strong reservations on how anything is phrased :)